.github/workflows/pr-welcome-msg.yml

```diff
@@ -19,9 +19,9 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             ## **Thank you for contributing to the Leapp project!**
-            Please note that every PR needs to comply with the [Leapp Guidelines](https://leapp.readthedocs.io/en/latest/contributing.html#) and must pass all tests in order to be mergeable.
-            If you want to request a review or rebuild a package in copr, you can use following commands as a comment:
-            - **review please** to notify leapp developers of review request
+            Please note that every PR needs to comply with the [Leapp Guidelines](https://leapp.readthedocs.io/en/latest/contributing.html#) and must pass all tests in order to be mergable.
+            If no reviewer reacts in a couple of days from PR creation add comment to this PR and tag `@oamg/developers` team.
```

Comment:
I'd rather keep it simple and remove the "if no reviewer reacts in ..." completely - just `To request a review please add a comment and tag @oamg/developers` should do imho.